src/test/java/edu/harvard/iq/dataverse/api/UsersIT.java

```diff
@@ -206,15 +207,15 @@ public void testMergeAccounts(){
         String aliasInOwner = "groupFor" + dataverseAlias;
         String displayName = "Group for " + dataverseAlias;
         String user2identifier = "@" + usernameConsumed;
+        String target2identifier = "@" + targetname;
         Response createGroup = UtilIT.createGroup(dataverseAlias, aliasInOwner, displayName, superuserApiToken);
         createGroup.prettyPrint();
         createGroup.then().assertThat()
                 .statusCode(CREATED.getStatusCode());
 
-        String groupIdentifier = JsonPath.from(createGroup.asString()).getString("data.identifier");
+        // String groupIdentifier = JsonPath.from(createGroup.asString()).getString("data.identifier");
```

Comment:
I suggest to remove code that is no longer needed, instead of leaving it commented out. The codebase has lots of commented code already, which is very rarely removed.